services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -35,18 +35,15 @@ export default class CurrentSubscription extends React.Component {
 
   getPaymentMethod() {
     const { subscriptionStatus, authorityLevel, subscriptionType, } = this.props
-    const { lastFour, } = this.state
 
-    if (subscriptionStatus && subscriptionStatus.payment_method === 'Credit Card' && lastFour && authorityLevel) {
+    if (subscriptionStatus && subscriptionStatus.payment_method === 'Credit Card' && authorityLevel) {
```

Comment:
This isn't your code, so feel free to punt for now, but we should consider using more constants for strings that are used for logic in the payments code, e.g. `'Credit Card'`.

Reply:
I would agree.  I'm planning on addressing this in one of the future stripe PRs.